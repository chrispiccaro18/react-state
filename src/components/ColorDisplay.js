import React from 'react';
import PropTypes from 'prop-types';
import style from '../index.css';

function ColorDisplay({ color }) {
  return <div className={style[color]}></div>;
}

ColorDisplay.propTypes = {
  color: PropTypes.string.isRequired
};

export default ColorDisplay;
