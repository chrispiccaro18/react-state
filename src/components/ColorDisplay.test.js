import React from 'react';
import { shallow } from 'enzyme';
import ColorDisplay from './ColorDisplay';

describe('ColorDisplay component', () => {
  it('renders a ColorDisplay', () => {
    const wrapper = shallow(<ColorDisplay color="red"/>);
    expect(wrapper).toMatchSnapshot();
  });
});
