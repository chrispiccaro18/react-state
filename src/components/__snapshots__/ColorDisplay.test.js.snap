// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorDisplay component renders a ColorDisplay 1`] = `
<div
  className="red"
/>
`;
